ocaml/testsuite/tests/typing-layouts-float64/stdlib_float_u_alpha.ml

```diff
@@ -0,0 +1,269 @@
+(* TEST
```

Comment:
Very pleasant reading here. I had wanted to create a whiz-bang QuickCheck clone to do this, but you've done something simpler and quicker, and yet still nicely readable. :+1:

Reply:
Er.... that's a bit embarrassing, really. I had no sense of familiarity. :grimacing: In any case, please trust that I wasn't trying to compliment myself!